wodles/gcloud/buckets/bucket.py

```diff
@@ -245,6 +248,12 @@ def process_data(self):
                             new_creation_time = current_creation_time
 
                         processed_files.append(blob)
+
+                    elif self.reparse:
+                        self.logger.info(f'File previously processed, but reparse flag set: {blob.name}')
```

Comment:
Place this message outside of the for loop, as you did with CloudWatch Logs. It just adds unnecessary verbosity to the module and it's not correct, because using the `--reparse` flag you might process a blob that you have never processed before.